packages/core/src/components/sections/ProductTiles/ProductTiles.tsx

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useRef } from 'react'
 import { useInView } from 'react-intersection-observer'
 
-import type { ProductsQueryQueryVariables } from '@generated/graphql'
+import type { BrowserProductsQueryQueryVariables } from '@generated/graphql'
```

Comment:
I don't think so because is automatically generated from `yarn generate` from the `BrowserProductsQuery` that needs to have `Query` in the end of the name.